src/pkg/caendr/caendr/api/phenotype.py

```diff
@@ -28,7 +28,7 @@ def query_phenotype_metadata(is_bulk_file=False, include_values=False, species:
     # Query by species
     if species is not None:
       if species in Species.all().keys():
```

Comment:
consider renaming `species` to `species_names` for semantic alignment with the next line